lms/templates/courseware/progress.html

```diff
@@ -171,13 +174,10 @@ <h2 class="title">${certificate_data.title | h}</h2>
                     ${section.format | h}
 
                     %if section.due is not None:
-                      <%
-                          formatted_string = get_time_display(section.due, course.due_date_display_format, coerce_tz=settings.TIME_ZONE_DISPLAYED_FOR_DEADLINES)
-                          due_date = '' if len(formatted_string)==0 else _(u'due {date}').format(date=formatted_string)
-                      %>
-                      <em>
-                      ${due_date | h}
-                      </em>
+                        <%
+                           data_string = _('due {date}')
```

Comment:
Same comment about not creating a temporary variable here.  It would read easier to just put the text into the next line.